sw/airborne/modules/multi/tcas.h

```diff
@@ -54,13 +57,22 @@ extern void tcas_init(void);
 extern void tcas_periodic_task_1Hz(void);
 extern void tcas_periodic_task_4Hz(void);
 
-#define CallTCAS() { if (tcas_status == TCAS_RA) v_ctl_altitude_setpoint = tcas_alt_setpoint; }
+extern void callTCAS(void);
 
-#define ParseTcasResolve() { \
-    if (DL_TCAS_RESOLVE_ac_id(dl_buffer) == AC_ID) { \
-      uint8_t ac_id_conflict = DL_TCAS_RESOLVE_ac_id_conflict(dl_buffer); \
-      tcas_acs_status[the_acs_id[ac_id_conflict]].resolve = DL_TCAS_RESOLVE_resolve(dl_buffer); \
-    } \
+static inline void parseTcasResolve(void)
```

Comment:
IMHO it is nicer to make these real functions (they are not speed critical) and move them to the c file. That way we can also minimize the includes needed in the header (only includes that are needed for declarations of things used in the header should be there).
That way we get less coupling and recompilation is faster.